editor/src/core/shared/uid-utils.ts

```diff
@@ -92,6 +95,10 @@ export function generateConsistentUID(
   existingIDs: Set<string>,
   possibleStartingValue: string,
 ): string {
+  if (COMMENT_FLAG_UIDS.current.has(possibleStartingValue)) {
+    return possibleStartingValue
+  }
```

Comment:
Is this code needed? I don't think it is if we're returning the value in `parseUID` below?

Reply:
prooobably the whole `COMMENT_FLAG_UIDS` is not needed anymore 🧐 running the tests to see if it's true

Reply:
Oh I think the problem might stem from this function (that looks to be very incorrect): <https://github.com/concrete-utopia/utopia/blob/b00117d11559bac9fc4c4301bac95b63700257fc/editor/src/core/shared/uid-utils.ts#L257-L262>

I'm not sure how that `includes` line came to be, but that will surely always return `true` 😀

Reply:
found the problem, sec

Reply:
https://github.com/concrete-utopia/utopia/pull/3443/commits/5244efc5a360189ed4d625db87ef1459d3c5eefd

Reply:
Thanks for fixing this!